src/main/java/be/seeseemelk/mockbukkit/block/state/BannerMock.java

```diff
@@ -25,15 +25,13 @@ public class BannerMock extends TileStateMock implements Banner
 	public BannerMock(@NotNull Material material)
 	{
 		super(material);
-		if (!Tag.BANNERS.isTagged(material))
-			throw new IllegalArgumentException("Cannot create a Banner state from " + material.name());
+		checkType(material, Tag.BANNERS); // Paper doesn't have the banners tag in the MaterialTags class (yet)
```

Comment:
This comment can be removed
```suggestion
		checkType(material, Tag.BANNERS);
```